src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <div class="img"></div>
+
+      <div class="text">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product_code">Product code: 195434</div>
+
+      <div class="stars">
+        <ul class="stars__stars_4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <div class="stars__review">Reviews: 5</div>
+      </div>
+
+      <div class="price">
+        <div class="price__text">Price:</div>
+        <div class="price__num">$2,199</div>
+      </div>
+
+      <div
+        class="button"
```

Comment:
You should have only 2 BEM block. One from the previous task `stars`.
And `card`, inside card elements have classes `card__price`, `card__button`, `card__title`, etc